tests/global-teardown.ts

```diff
@@ -2,6 +2,8 @@ import Strapi from "@strapi/strapi";
 import { cleanUpDatabase } from "../init/common";
 
 async function globalTeardown() {
+    // need to keep this line, otherwise github annotations will not work
```

Comment:
This is very weird. Effectively Playwright reporter needs this variable, and I don't know how, but somehow printing this out here helps to process annotations report correctly. See https://github.com/microsoft/playwright/blob/main/packages/playwright/src/reporters/github.ts#L124
Compare logs in Github action without this line: https://github.com/dev-launchers/strapiv4/actions/runs/9637422370/job/26576618409?pr=192#step:6:96 
(error is not recognized as an error) and with this log
https://github.com/dev-launchers/strapiv4/actions/runs/9637479783/job/26576801796?pr=192#step:6:97
Maybe that teardown happens too fast or somethings, probably any logging would help, but I just found it by chance 🤷‍♂️